mantidproject/mantid
# Make elwin tab mvp

**Description of work.**
This PR changes the elwin tab in inelastic data manipulation to be MVP format.

**To test:**

1. Open indirect data Manipulation and go to the elwin tab.
2. Load a set of data and click run without cahngign any settigns, check the output is appropriate.
3. change the integretion range and run again, check that the output is created with the new range, (check output history)
4. enable background and run, check that the history shows background
5. enable normalize and run, check again the history.


Fixes #34666 

*This does not require release notes* because It is an entirly internal change and has no outward effect on users.


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
